sync/awsdatabricks.py

```diff
@@ -76,6 +110,7 @@ def get_cluster(cluster_id: str) -> Response[dict]:
     return Response(result=cluster)
 
 
+# TODO - Databricks configuration documentation
 def create_prediction_for_run(
     run_id: str, plan_type: str, compute_type: str, project_id: str = None
```

Comment:
I defined the `plan_type` and `compute_type` enums, but haven't updated these functions to use them everywhere, yet... this diff is already getting a little large